deposit/src/main/java/edu/unc/lib/deposit/normalize/Simple2N3BagJob.java

```diff
@@ -126,6 +130,27 @@ private void populateSimple(Model model, Resource primaryResource, String alabel
 			failJob("Failed to find upload file for simple deposit: " + filename,
 					contentFile.getAbsolutePath());
 		}
+		
+		String checksum = null;
+		String fullPath = contentFile.toString();
+		
+		try {
+			checksum = DigestUtils.md5Hex(new FileInputStream(fullPath));
+			
+			PremisLogger premisDepositLogger = getPremisLogger(getDepositPID());
+			PremisEventBuilder premisDepositEventBuilder = premisDepositLogger.buildEvent(Premis.MessageDigestCalculation);
```

Comment:
This is just a style and readability thing, but the eventBuilder generally doesn't need to be visible as a local variable if you aren't going to need to use it past a single chain of methods, like so:

```java
Resource event = premisDepositLogger.buildEvent(Premis.MessageDigestCalculation)
        .addEventDetail("Checksum for file is {0}", checksum)
        .addSoftwareAgent(SoftwareAgent.depositService.getFullname())
        .create();
```